infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/DefaultClusterManager.java

```diff
@@ -181,57 +229,161 @@ public TopologyConfigurationMsg queryTopologyConfig(boolean useCached) {
     }
 
     /**
-     * Enforce one of the standby Cluster's to become the new active cluster and current active to become standby
+     * Create a new topology config, which changes one of the standby as the active,
+     * and active as standby. Data should flow in the reverse direction.
```

Comment:
can you annotate it with @TestOnly or add it in the comment that this is only used by tests?

Reply:
Actually the entire DefaultClusterManager is test-only.